### PR TITLE
Faking availableReplicas to enable usage of the gateway UI

### PR DESCRIPTION
**Pull request by jgnagy**
I'm no Go developer so feel free to make fun of anything I did wrong here. I was able to run `make build` and it seems to have succeeded but I wasn't sure how to run unit tests.

@alexellis and @kenfdev please let me know if I need to do this differently.

### CODE REVIEW

**Comment by alexellis** [comment]
You have 4 commits now, so do: `git reset HEAD~4`  then git add and commit -s followed by a force push. That should sort things out.

Alex

File: handlers/replicas.go [hunk 1]
```diff
@@ -85,6 +85,8 @@ func MakeReplicaReader(client rancher.BridgeClient) VarsHandler {
 			return
 		}
 
+		found.AvailableReplicas = found.Replicas
```
Comment (kenfdev) [inline handlers/replicas.go:88]: Seems reasonable. But could you leave a comment about faking the information? Thanks ;)

**Review by kenfdev — CHANGES_REQUESTED** [review]
Thank you @jgnagy for your contribution. This change seems reasonable since currently faas-rancher and the new gateway can't use the gateway UI.

Few things I want to make sure:

* How did you update the vendor file? This repo uses [rancher/trash](https://github.com/rancher/trash) for vendors. (not `dep` IIRC).
* Could you provided a little more information about how you tested this?

**Comment by alexellis** [comment]
Let's switch to dep Ken?

**Comment by kenfdev** [comment]
> Let's switch to dep Ken?

I'm not against that and that's a reasonable change, too. 
Unfortunately, I don't have the time to do it myself but if @jgnagy or someone in the community could take it, I'm more than happy to review it.

**Comment by jgnagy** [comment]
#7 is a **much** better approach. This PR is no longer necessary.

**Pull request closed without merging**